.github/workflows/override_version_bump_pr_checks.yml

```diff
@@ -6,12 +6,15 @@ on:
 jobs:
   override_checks:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      statuses: write # Update status checks
 
     steps:
-      - name: Fail if branch is not version-bump PR
-        if: ${{ !startsWith(github.ref, 'refs/heads/version-bump-') }}
+      - name: Fail if branch is not version-bump PR or bot PR
+        if: ${{ !startsWith(github.ref, 'refs/heads/version-bump-') && !startsWith(github.ref, 'refs/heads/bot/') }}
```

Comment:
Fixes a minor annoyance that the version-bump PRs can't be merged unless you rebase/force-merge, because the checks don't automatically run